keras/distribution/distribution_lib_test.py

```diff
@@ -3,15 +3,10 @@
 import os
 from unittest import mock
 
-import jax
 import numpy as np
-import pytest
 
 from keras import backend
-from keras import layers
-from keras import models
 from keras import testing
-from keras.backend import distribution_lib as backend_dlib
 from keras.distribution import distribution_lib
 
 if backend.backend() == "jax":
```

Comment:
Let's move this section to the jax backend test.

Reply:
It's alreadyh in the jax test.
Removed this from the general distribution test.